ipatests/test_integration/test_replica_promotion.py

```diff
@@ -1029,6 +1029,9 @@ def test_hidden_replica_backup_and_restore(self):
             stdin_text=dirman_password + '\nyes'
         )
 
+        # wait for the replica to be available
```

Comment:
Is it possible for `ipa-restore` to exit successfully while IPA services are not in the RUNNING state?

Reply:
Good catch, thanks.